automated_analysis.py

```diff
@@ -465,4 +460,47 @@ def set_survey_percentages(survey_counts, total_survey_counts):
         fig.write_image(f"{automated_analysis_output_dir}/graphs/{plan.raw_field}_by_gender_normalised.png",
                         scale=IMG_SCALE_FACTOR)
 
+    # Export safe to share raw messages for each episode to share with WUSC
+    log.info("Exporting safe to share raw messages for each episode...")
+    safe_to_share_messages = []  # of dict of code_string_value to raw messages
+    no_of_dns_messages = 0
+    for plan in PipelineConfiguration.RQA_CODING_PLANS[5:]: #loop through episode 6 -> because previous episodes had been manually processed
```

Comment:
```suggestion
    for plan in PipelineConfiguration.RQA_CODING_PLANS[5:]:  # Loop through episode 6 -> because previous episodes had been manually processed
```

Reply:
updated thanks